README.md

```diff
@@ -31,26 +31,26 @@ export DB_PROTOCOL
 DB_PROTOCOL=postgres
 ```
 
-To do this, it relies on a config file ("dbcli.yml") which teaches it
+For details on the potential facts returned, see the
+[API reference](https://db-facts.readthedocs.io/en/published_docs/db_facts.html#db_facts.DBFacts).
+
+`db-facts` relies on a config file ("dbcli.yml") which teaches it
 how to parse the user-friendly coordinates.  Much of the heavy lifting
 in the parsing part is done
 by
 [jinja_context.py](https://github.com/bluelabsio/db-facts/blob/master/db_facts/jinja_context.py),
 which sets some variables and functions that can be used in jinja
 templates within the config file.
 
-This is part of the suite of programs which allow a user to type in
-things like `db redshift` and connect via their own credentials to the
-configured database named 'redshift'.  Other parts of this chain can
-be found in the [ws-scripts](https://github.com/bluelabsio/ws-scripts)
-repo.
-
```

Comment:
Whoops!  We still had a reference to an internal repo.  Getting this stuff packaged up and open sourced is in the backlog, but not on the critical path to get 1.0 of records-mover out.